fireblocks_sdk/api_types.py

```diff
@@ -317,31 +316,209 @@ class TimePeriod(str, Enum):
     DAY = "DAY"
     WEEK = "WEEK"
 
+class StellarRippleCreateParams:
+    def __init__(
+        self,
+        issuerAddress: Optional[str] = None
+    ):
+        self.issuerAddress = issuerAddress
+
+    def serialize(self) -> dict:
+        obj = {}
+
+        if self.issuerAddress:
+            obj.update({'issuerAddress': self.issuerAddress})
 
-class IssueTokenRequest:
-    symbol: str
-    name: str
-    blockchain_id: str
-    eth_contract_address: Optional[str]
-    issuer_address: Optional[str]
-    decimals: int
+        return obj
 
+class Parameter:
+    def __init__(
+        self,
+        name: str,
+        type: str,
+        internalType: str,
+        description: Optional[str] = None,
+        components: Optional[List['Parameter']] = None
+    ):
+        self.name = name
+        self.type = type
+        self.internalType = internalType
+        self.description = description
+        self.components = components
+    
     def serialize(self) -> dict:
         obj = {
+            'name': self.name,
+            'type': self.type,
+            'internalType': self.internalType,
+        }
+
+        if self.description:
+            obj.update({'description': self.description})
+        if self.components:
+            obj.update({'components': self.components})
+
+        return obj
+
+class ParameterWithValue(Parameter):
+    def __init__(
+        self,
+        name: str,
+        type: str,
+        internalType: str,
+        value: Union[str, int, float, bool],
+        description: Optional[str] = None,
+        components: Optional[List['Parameter']] = None
+    ):
+        super().__init__(name, type, internalType, description, components)
+        self.value = value
+    
+    def serialize(self) -> dict:
+        obj = super().serialize()
+        obj.update({'value': self.value})
+        
+        return obj
+
+class EVMTokenCreateParams:
+    def __init__(
+        self,
+        contractId: str,
+        constructorParams: Optional[List[ParameterWithValue]] = []
```

Comment:
in python we shouldn't put a default value for a mutable object as it will be passed between different function calls, you should init it as None and check and if it is None (meaning the user didn't supply their own list) you will init it to an empty list